### ISSUE TITLE
Training args with COCO dataset

### ISSUE THREAD

**Issue by hwan6615** [DavidHuji/CapDec#20]
I'm currently trying to reproduce the results using the COCO dataset, but I'm unable to achieve the performance reported in the paper. 

Would it be possible to know the arguments used for training?

**Comment by DavidHuji** [comment]
Thank you for your interest in this repository! I have set the best hyperparameters as the default values in argparse. Which arguments are missing that you need assistance with? What performance did you achieve, and how does it compare to the results reported in the paper?

**Comment by hwan6615** [comment]
Thank you for your good paper and response. 

The experiments were conducted using the COCO dataset.

First, the arguments and performance before normalization are as follows. 
![화면 캡처 2024-12-15 054411](https://github.com/user-attachments/assets/22ac3d1c-2a8b-426d-8b73-0c7f49974786)
![image](https://github.com/user-attachments/assets/66440b10-1df6-4afa-8bc6-70767f5cf4e4)




When normalization was not performed, the performance was very low. Performance is best at 1 epoch, and as training progresses, performance declines.


In contrast, when normalization was performed, the performance improved, but compared to the performance in the paper, it is considerably low, and BLEU-4 is particularly low. 
![image](https://github.com/user-attachments/assets/4c2b7944-c161-4a54-be0e-97a619736a47)
![image](https://github.com/user-attachments/assets/2b14d30b-1e88-4064-8a73-8da7e4acc079)




Do you think there might be any issues, and is there any additional information you would like?

**Comment by DavidHuji** [comment]
Looks like the batch size is a bit bigger but i do not think this is the issue. Did you evaluate on the last epoch? Maybe try earlier ones where the loss is already low as much as in the last epoch (i.e. the 7th rather than the 10th).

**Comment by hwan6615** [comment]
I conducted evaluations for all 10 epochs, and this represents the best performance among them. As mentioned above, without normalization, performance decreases as epochs progress. When normalization is applied, performance improves with each epoch, but falls significantly short of the performance reported in the original paper.

**Comment by DavidHuji** [comment]
Oh I see, I missed that earlier. I think then trying a smaller bs as in the default parameters might be helpful (or at least increase the lr instead or the amount of epochs) because it sound like the training was just not long enough (when the bs is x4 bigger, there are x4 less weights updates).

**Comment by hwan6615** [comment]
Thank you for your response. I will proceed with an experiment.
